src/geb/geb.lisp

```diff
@@ -267,7 +345,7 @@ This is the morphism part of the product functor Geb x Geb → Geb"
   "Id")
 
 (defmethod maybe ((obj <substobj>))
-  "I recursively add maybe terms to all [\\<SBUSTOBJ\\>][class] terms,
+  "I recursively add maybe terms to all [\\<SUBSTOBJ\\>][class] terms,
```

Comment:
I don't se the diff here, did an encoding of the word get better, did I fat finger a unicode character here by accident?

Reply:
I thought this was a typo. I'll remove this change